packages/forms-web-app/src/controllers/full-appeal/submit-appeal/new-documents.js

```diff
@@ -3,30 +3,50 @@ const { createOrUpdateAppeal } = require('../../../lib/appeals-api-wrapper');
 const {
   VIEW: {
     FULL_APPEAL: {
-      OTHER_SUPPORTING_DOCUMENTS,
       NEW_DOCUMENTS,
+      OTHER_SUPPORTING_DOCUMENTS,
       TASK_LIST,
       PLANNING_OBLIGATION_PLANNED,
+      PLANNING_OBLIGATION_DOCUMENTS,
+      PLANNING_OBLIGATION_DEADLINE,
+      DRAFT_PLANNING_OBLIGATION,
     },
   },
 } = require('../../../lib/full-appeal/views');
+const {
+  constants: { PLANNING_OBLIGATION_STATUS_OPTION },
+} = require('@pins/business-rules');
 const { COMPLETED } = require('../../../services/task-status/task-statuses');
 
 const sectionName = 'appealDocumentsSection';
 const taskName = 'supportingDocuments';
-const backLink = `/${PLANNING_OBLIGATION_PLANNED}`;
+
+const getBackLink = (planningObligation, status) => {
+  if (planningObligation) {
```

Comment:
I had to wrap switch logic in an if statement because if a user initially selects 'yes' for planning obligation and then. say, 'draft', the 'draft' status will not disappear if the user goes back and changes answer to 'no' for planning obligation (so default option on switch logic won't work). Therefore, the logic checks the boolean for whether or not they've selected yes or no before checking the status of the planning obligation (if relevant).